2022-01-20 01:46:21,644 INFO    Thread-2  :40157 [wandb_setup.py:_flush():71] setting env: {'entity': 'amazonproject', 'project': 'Amazon-Reviews', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project', 'sweep_id': 'hp8o2bs7', 'run_id': 'xb5ramvs', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/sweep-hp8o2bs7/config-xb5ramvs.yaml'}
2022-01-20 01:46:21,644 INFO    Thread-2  :40157 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 01:46:21,645 INFO    Thread-2  :40157 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220120_014621-xb5ramvs/logs/debug.log
2022-01-20 01:46:21,645 INFO    Thread-2  :40157 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220120_014621-xb5ramvs/logs/debug-internal.log
2022-01-20 01:46:21,645 INFO    Thread-2  :40157 [wandb_init.py:init():404] calling init triggers
2022-01-20 01:46:21,645 INFO    Thread-2  :40157 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 200, 'drop_out': 0.35, 'epochs': 5, 'lr': 0.01, 'num_workers': 4, 'optimizer': 'adam'}
config: {}
2022-01-20 01:46:21,645 INFO    Thread-2  :40157 [wandb_init.py:init():460] starting backend
2022-01-20 01:46:21,646 INFO    Thread-2  :40157 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-20 01:46:21,662 INFO    Thread-2  :40157 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 01:46:21,675 INFO    Thread-2  :40157 [backend.py:ensure_launched():221] started backend process with pid: 40165
2022-01-20 01:46:21,678 INFO    Thread-2  :40157 [wandb_init.py:init():469] backend started and connected
2022-01-20 01:46:21,683 INFO    Thread-2  :40157 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 200, 'drop_out': 0.35, 'epochs': 5, 'lr': 0.01, 'num_workers': 4, 'optimizer': 'adam'}
2022-01-20 01:46:21,689 INFO    Thread-2  :40157 [wandb_init.py:init():533] updated telemetry
2022-01-20 01:46:21,729 INFO    Thread-2  :40157 [wandb_init.py:init():563] communicating current version
2022-01-20 01:46:23,576 INFO    Thread-2  :40157 [wandb_init.py:init():568] got version response 
2022-01-20 01:46:23,576 INFO    Thread-2  :40157 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 01:46:23,871 INFO    Thread-2  :40157 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 01:46:28,882 INFO    Thread-2  :40157 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 01:46:28,883 INFO    Thread-2  :40157 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-20 01:46:28,883 INFO    Thread-2  :40157 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-20 01:46:28,883 ERROR   Thread-2  :40157 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-20 01:46:28,888 INFO    Thread-2  :40157 [wandb_init.py:init():633] run started, returning control to user process
